web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -386,10 +395,20 @@ def find_user_type(self):
             return self.subject.user_type
 
     @property
-    def additional_industry_ids(self):
+    def additional_industries(self):
         if hasattr(self.subject, "additional_industries"):
-            return self.subject.additional_industries.values_list(
-                "id", flat=True)
+            industries = self.subject.additional_industries
+            result = []
```

Comment:
seems like a lot of this is duplicated from https://github.com/masschallenge/impact-api/pull/73/files#diff-69930414b4eeea3bab51315196d2df4eR103

it could be consolidated.

Reply:
Yeah, CodeClimate caught that as well.  Duplication removed.

Reply:
Also fixed the Code Complexity issue reported by CodeClimate.